.gitignore

```diff
@@ -5,6 +5,9 @@ __pycache__/
 nmma.egg-info/
 .DS_Store
 
+# development
+.vscode/
+
 # models
 svdmodels/Bu2019lm_lbol.pkl
```

Comment:
Can we modify the ignore to ignore everything in this folder now?

Reply:
the change to gitignore should do that now, no?

Reply:
Ah I meant in svdmodels. I noticed that it still calls out individual files

Reply:
ah okay, I'll open that as a separate pr then

Reply:
Cool